### ISSUE TITLE
engines configurations

### ISSUE THREAD

**Issue by gaboesquivel** [mattdesl/module-best-practices#10]
@mattdesl I believe using engines configurations in package.json could also be considered a best practice. `engines` and `engineStrict`.  What do you think?

**Comment by mattdesl** [comment]
:+1:

**Comment by mattdesl** [comment]
Actually – I think it really depends on the module/CLI/tool you are building. :smile: I hate when I install something that gives me an error, but it works fine in all versions I'm using.

**Comment by gaboesquivel** [comment]
yeah I hear you... but I think that´s a maintenance problem...
In theory is good to specify node version ranges, what if a major release breaks it?  you should explicitly tell it works on that new release after testing.

**Comment by mattdesl** [comment]
Another approach is just to have tests that run in Travis on multiple node versions - this way users can see what versions it will work with.

**Comment by gaboesquivel** [comment]
you are right, that´s better. I think that deserves it´s place in the testing section.

**Comment by gaboesquivel** [comment]
another reason to include `engines` field as best practice is to facilitate node version switching. 
Eg. Using [nodengine](https://github.com/Kikobeats/nodengine)
